cmd/substation/fmt.go

```diff
@@ -0,0 +1,100 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/google/go-jsonnet/formatter"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	rootCmd.AddCommand(fmtCmd)
+	fmtCmd.PersistentFlags().BoolP("write", "w", false, "write result to (source) file instead of stdout")
+	fmtCmd.PersistentFlags().BoolP("recursive", "R", false, "recursively format all files")
+}
+
+var fmtCmd = &cobra.Command{
+	Use:   "fmt [path]",
+	Short: "Format Jsonnet files",
+	Long: `'substation fmt' formats Jsonnet files.
+It prints the formatted output to stdout by default.
+Use the --write flag to update the files in-place.
+
+The command can format a single file or a directory.
+Use the --recursive flag to format all files in a directory and its subdirectories.
+
+Supported file extensions: .jsonnet, .libsonnet`,
+	Example: `  substation fmt config.jsonnet
+  substation fmt -w config.jsonnet
+  substation fmt -R /path/to/configs
+  substation fmt -w -R /path/to/configs`,
+	Args: cobra.MaximumNArgs(1),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		// Default to current directory if no path is provided
+		path := "."
+		if len(args) > 0 {
+			path = args[0]
+		}
+
+		write, _ := cmd.Flags().GetBool("write")
+		recursive, _ := cmd.Flags().GetBool("recursive")
```

Comment:
What's the impact of not catching these errors?

Reply:
Good call! No actual reasoning for skipping em. We should propagate these up.